utils/promis.js

```diff
@@ -0,0 +1,578 @@
+const CryptoJS = require('crypto-js');
+
+const generatePromisAuthToken = () => {
+
+    // const { getSecret } = require('./shared');
```

Comment:
not needed?

Reply:
this will be needed and is a reminder for me... I want to get people testing this in dev, but before this goes to prod we need to change from the testing URL / token to a prod URL / token (which I will keep in a secret)